src/scf-release/src/acceptance-tests-brain/test-scripts/testutils.rb

```diff
@@ -185,6 +185,13 @@ def cf_curl(*args)
     JSON.parse output
 end
 
+def wait_for_statefulset(namespace, statefulset, sleep_duration=10)
```

Comment:
How about writing this like https://github.com/SUSE/scf/blob/develop/src/scf-release/src/acceptance-tests-brain/test-scripts/testutils.rb#L160-L168?

```ruby
def wait_for_statefulset_ready(name, namespace, timeout=300)
    run %W(
        kubectl wait statefulset/#{name}
            --for condition=Ready
            --namespace #{namespace}
            --timeout #{timeout}s
    ).join(" ").chomp
end
```